### PR TITLE
Reduce CRA interval to 5 days

### PR DESCRIPTION
**Pull request by EdNoepel**
## Description

Previously, Claimable Reserve Auctions (CRAs) could be kicked no sooner than 2 weeks after the last auction completed.  This change reduces that interval to 5 days (120 hours), measured from the kick of the previous CRA.

## Purpose

Reduce the profitability of attacks which aim to steal reserves by intentionally pushing bad debt onto the pool..

## Impact

No contract size or gas changes anticipated.

## Tasks

- [x] Changes to protocol contracts are covered by unit tests executed by CI.
- [x] Protocol contract size limits have not been exceeded.
- [ ] Gas consumption for impacted transactions have been compared with the target branch, and nontrivial changes cited in the _Impact_ section above.
- [x] Scope labels have been assigned as appropriate.
- [x] Invariant tests have been manually executed as appropriate for the nature of the change.

### CODE REVIEW

**Comment by EdNoepel** [comment]
Invariant tests found several failures, which can be reproduced in `develop` as well.  Best wait until PR #955 is merged to investigate.

**Review by prat-gpt — APPROVED** [review]
LGTM

**Review by ith-harvey — APPROVED** [review]
LGTM

File: src/libraries/external/KickerActions.sol
Comment (mattcushman) [inline src/libraries/external/KickerActions.sol]: We should review if `5 days` is still too much to accomplish what we set out to accomplish with this.  The `72 hours` is another `3 days`, so if someone can effectively block for `8 days` then they can prevent the sweeping of the auction withing `144 hours` which is `6 days`.

Comment (EdNoepel) [reply src/libraries/external/KickerActions.sol]: Per this morning's discussion, will update source to skip 120 hours from kick (3 days for auction + 2 day interval).

Comment (EdNoepel) [reply src/libraries/external/KickerActions.sol]: Updated and marked for re-review.

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Review by ith-harvey — APPROVED** [review]
LGTM

**Review by mattcushman — APPROVED** [review]
lgtm

**Review by grandizzy — APPROVED** [review]

File: src/libraries/external/KickerActions.sol
Comment (k1rill-fedoseev) [inline src/libraries/external/KickerActions.sol]:
```suggestion
        // check that at least five days have passed since the last reserve auction was kicked
```

**Review by k1rill-fedoseev — APPROVED** [review]

**Review by dmitriia — COMMENTED** [review]
Ok, but some cleanup can be done:

1. Descriptions

`takeReserves()` reverts if auction is not yet open or already closed:

https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/libraries/external/TakerActions.sol#L275-L276

```diff
     *  @dev    === Reverts on ===
-    *  @dev    not kicked or `72` hours didn't pass `NoReservesAuction()`
+    *  @dev    not kicked or `72` hours passed `NoReservesAuction()`
```

Both `kickReserveAuction()` revert descriptions can be enriched:

https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/libraries/external/KickerActions.sol#L206-L216

```diff
    /**
     *  @notice See `IPoolKickerActions` for descriptions.
     *  @dev    === Write state ===
     *  @dev    update `reserveAuction.unclaimed` accumulator
     *  @dev    update `reserveAuction.kicked` timestamp state
     *  @dev    === Reverts on ===
     *  @dev    no reserves to claim `NoReserves()`
+    *  @dev    5 days not passed `ReserveAuctionTooSoon()`     
     *  @dev    === Emit events ===
     *  @dev    - `KickReserveAuction`
     */
    function kickReserveAuction(
```

https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/base/Pool.sol#L391-L402

```diff
    /**
     *  @inheritdoc IPoolKickerActions
     *  @dev    === Write state ===
     *  @dev    increment `latestBurnEpoch` counter
     *  @dev    update `reserveAuction.latestBurnEventEpoch` and burn event `timestamp` state
     *  @dev    === Reverts on ===
+    *  @dev    no reserves to claim `NoReserves()`
     *  @dev    5 days not passed `ReserveAuctionTooSoon()`
     *  @dev    unsettled liquidation `AuctionActive()`
     *  @dev    === Emit events ===
     *  @dev    - `KickReserveAuction`
     */
    function kickReserveAuction() external override nonReentrant {
```

2. Unused errors:

https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/libraries/external/TakerActions.sol#L120-L122

```diff
-   error NoReserves();
    error NoReservesAuction();
-   error ReserveAuctionTooSoon();
```

https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/libraries/external/KickerActions.sol#L98-L101

```diff
    error InsufficientLiquidity();
-   error InsufficientLP();
-   error InvalidAmount();
    error NoReserves();
```

**Comment by grandizzy** [comment]
> Ok, but some cleanup can be done:
> 
>     1. Descriptions
> 
> 
> `takeReserves()` reverts if auction is not yet open or already closed:
> 
> https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/libraries/external/TakerActions.sol#L275-L276
> 
> ```diff
>      *  @dev    === Reverts on ===
> -    *  @dev    not kicked or `72` hours didn't pass `NoReservesAuction()`
> +    *  @dev    not kicked or `72` hours passed `NoReservesAuction()`
> ```
> 
> Both `kickReserveAuction()` revert descriptions can be enriched:
> 
> https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/libraries/external/KickerActions.sol#L206-L216
> 
> ```diff
>     /**
>      *  @notice See `IPoolKickerActions` for descriptions.
>      *  @dev    === Write state ===
>      *  @dev    update `reserveAuction.unclaimed` accumulator
>      *  @dev    update `reserveAuction.kicked` timestamp state
>      *  @dev    === Reverts on ===
>      *  @dev    no reserves to claim `NoReserves()`
> +    *  @dev    5 days not passed `ReserveAuctionTooSoon()`     
>      *  @dev    === Emit events ===
>      *  @dev    - `KickReserveAuction`
>      */
>     function kickReserveAuction(
> ```
> 
> https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/base/Pool.sol#L391-L402
> 
> ```diff
>     /**
>      *  @inheritdoc IPoolKickerActions
>      *  @dev    === Write state ===
>      *  @dev    increment `latestBurnEpoch` counter
>      *  @dev    update `reserveAuction.latestBurnEventEpoch` and burn event `timestamp` state
>      *  @dev    === Reverts on ===
> +    *  @dev    no reserves to claim `NoReserves()`
>      *  @dev    5 days not passed `ReserveAuctionTooSoon()`
>      *  @dev    unsettled liquidation `AuctionActive()`
>      *  @dev    === Emit events ===
>      *  @dev    - `KickReserveAuction`
>      */
>     function kickReserveAuction() external override nonReentrant {
> ```
> 
>     2. Unused errors:
> 
> 
> https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/libraries/external/TakerActions.sol#L120-L122
> 
> ```diff
> -   error NoReserves();
>     error NoReservesAuction();
> -   error ReserveAuctionTooSoon();
> ```
> 
> https://github.com/ajna-finance/contracts/blob/3f7525ae53ed4ded3b4e0f6c3355756d523c695b/src/libraries/external/KickerActions.sol#L98-L101
> 
> ```diff
>     error InsufficientLiquidity();
> -   error InsufficientLP();
> -   error InvalidAmount();
>     error NoReserves();
> ```

created https://github.com/ajna-finance/contracts/pull/1048 to address this

**Pull request merged**
